### PR TITLE
Add xml and json output format

### PR DESCRIPTION
**Pull request by arekzaluski**
## Why
It is very difficult to parse existing `txt` output. Addition of `xml` and `json` will allow the output to be easily parsable by other tools/extensions.

## Changes
- Add `xml` and `json` option to format output
It will print events and event statistics as xml/json file or on stdout.

## Example:
 ```
eventlist -f json -o output.json <logFile>
```

### CODE REVIEW

File: tools/eventlist/pkg/output/output.go [hunk 1]
```diff
@@ -553,9 +643,29 @@ func Print(filename *string, eventFile *string, evdefs map[uint16]scvd.Event,
...
+			if err == nil {
+				buf := bytes.NewBuffer(output)
+				_, err = fmt.Fprint(out, buf)
+				if err == nil {
+					err = out.Flush()
+				}
+			}
+		} else {
+			err = out.Flush()
+		}
```
Comment (arekzaluski) [inline tools/eventlist/pkg/output/output.go:668]: Very easily another formats can be added. For example `yaml`.

**Comment by JonatanAntoni** [comment]
@arekzaluski, could you address the lint warnings, please?

File: tools/eventlist/pkg/output/output_test.go
Comment (bgn42) [inline tools/eventlist/pkg/output/output_test.go:1]: I think that there are tests for formatType xml and json missing.

Comment (arekzaluski) [reply tools/eventlist/pkg/output/output_test.go:1]: Yes, I'll add tests for json and xml to this PR today

File: tools/eventlist/pkg/output/output_test.go [hunk 2]
```diff
@@ -574,7 +574,7 @@ func Test_eventProperty_getLast(t *testing.T) {
 	t.Parallel()
 
 	type fields struct {
-		values [16]eventStatistic
+		Values [16]eventStatistic
```
Comment (arekzaluski) [inline tools/eventlist/pkg/output/output_test.go:577]: I'll rename it back to small `values` as we do not need to expose it

File: tools/eventlist/pkg/output/output_test.go
Comment (arekzaluski) [reply tools/eventlist/pkg/output/output_test.go:1]: Tests added

File: tools/eventlist/pkg/output/output.go [hunk 3]
```diff
@@ -279,60 +320,87 @@ func (o *Output) buildStatistic(in *bufio.Reader, evdefs map[uint16]scvd.Event,
 	return eventCount
 }
 
-func (o *Output) printStatistic(out *bufio.Writer, eventCount int) error {
+func write(out *bufio.Writer, format string, a ...any) (n int, err error) {
```
[the remark this answers is not in the document]
Comment (arekzaluski) [reply tools/eventlist/pkg/output/output.go]: Renamed

**Review by bgn42 — APPROVED** [review]
Looks very good now.

**Pull request merged**